src/ch/ch_monitor.c

```diff
@@ -913,18 +917,40 @@ static int virCHMonitorReadProcessEvents(virCHMonitorPtr mon)
     return (events > 0) ? virCHMonitorProcessEvents(mon, events) : events;
 }
 
+#define MONITOR_TMP_FAIL_RETRIES 5
```

Comment:
For `MONITOR_TMP_FAIL_RETRIES` why the `TMP` in the name here?

Reply:
Just to denote that this is the number of retries that we do on any _**Temporary**_ failures in CH Monitor logic. If it is confusing, I can change the name or add a comment to clarify

Reply:
Added a comment to clarify!

Reply:
Ahh, that's probably fine without a comment.